roles/download/defaults/main.yml

```diff
@@ -115,7 +115,15 @@ flannel_cni_version: "v1.1.0"
 cni_version: "v1.1.1"
 weave_version: 2.8.1
 pod_infra_version: "3.7"
+
 cilium_version: "v1.12.1"
+# Enable Hubble without install
+cilium_enable_hubble: false
+# Enable Hubble install
+cilium_hubble_install: false
```

Comment:
Are there any use cases for enabling hubble without installing it?

In this main.yml, there is not such cases today.
`enabled: "{{ cilium_enable_hubble and cilium_hubble_install }}"` seems a little bit complex, and `enabled: "{{ cilium_enable_hubble }}` would be easier like the other options if possible.

Reply:
you're right, it really should be kept succinct here.

Reply:
@oomichi 😊 code has been updated ans PTAL.